### PR TITLE
Add falconapi cli

### PR DESCRIPTION
**Pull request by redhatrises**
1. Obviously if this is undesired, no worries.
2. Not tied to the name. Not fully a fan of it either. I thought `falconctl` would be cool but obviously conflicts. Thoughts around the existing name or maybe `falconapictl`?

### CODE REVIEW

**Comment by redhatrises** [comment]
So another name I came up for the cli is `csfapictl` or `csapictl`.

**Review by isimluk — APPROVED** [review]

**Comment by redhatrises** [comment]
@isimluk the recommendation from @shawndwells was to keep it at `falconapi` for consistency... so updated the PR to remove the `-`

**Comment by isimluk** [comment]
I am still little bit concerned about the amount of dependencies we bring to the dependency chain of simple library. Is there any way to avoid some of the deps?

File: go.mod [hunk 1]
```diff
@@ -9,6 +9,9 @@ require (
 	github.com/go-openapi/strfmt v0.20.2
 	github.com/go-openapi/swag v0.19.15
 	github.com/go-openapi/validate v0.20.2
+	github.com/pkg/errors v0.9.1
 	github.com/sirupsen/logrus v1.8.1
-	golang.org/x/oauth2 v0.0.0-20210113205817-d3ed898aa8a3
+	github.com/spf13/cobra v0.0.3
```
[the remark this answers is not in the document]
Comment (redhatrises) [reply go.mod]: No actually, I think my vscode tools choose it automatically for some weird reason or another.

**Comment by redhatrises** [comment]
> I am still little bit concerned about the amount of dependencies we bring to the dependency chain of simple library. Is there any way to avoid some of the deps?

Well..... I guess we could create a new repo.... or not have a robust cli framework.... or....?

**Pull request closed without merging**